packages/charts/chart-utilities/project.json

```diff
@@ -3,6 +3,6 @@
   "$schema": "../../../node_modules/nx/schemas/project-schema.json",
   "projectType": "library",
   "sourceRoot": "packages/charts/chart-utilities/src",
-  "tags": ["platform:web", "vNext", "charting"],
+  "tags": ["v8", "ships-bundle", "charting"],
```

Comment:
hmm its not good idea to "misuse" domain tags to tweak release groups. but if the code is more v8 like then it's correct from both ends.

Wouldn't it be better if charting had separate release groups completely , independent of core v8,v9 ? WDYT ?

Reply:
Actually, this specific package is unique where it is independent of v8 or v9.

https://github.com/microsoft/fluentui/issues/34423
Filed a bug to investigate how to handle this package better. I need to understand the release groups definition better. 
Please share any pointers for this.

Reply:
yeah good call, if it's agnostic it should contain probably both `v8` and `vNext` as we have in our migration packages for example.

Reply:
Will test and take up this work separately as part of above issue. Thanks